{"name": "Тест на создание двух одинаковых пользователей", "status": "passed", "description": "Создаём одного юзера,затем создаём второго юзера с таким же набором данных. Проверяем, что второй юзер не создался и текст ошибки: \"User already exists\". После проверки удаляем юзера", "steps": [{"name": "Создаём юзера", "status": "passed", "start": 1717572726066, "stop": 1717572726303}, {"name": "Создаём юзера", "status": "passed", "start": 1717572726303, "stop": 1717572726521}, {"name": "Удаляем юзера", "status": "passed", "start": 1717572726521, "stop": 1717572726977}], "start": 1717572726066, "stop": 1717572726977, "uuid": "3705e615-82e0-421f-8b50-6f06f437b4c2", "historyId": "d90981d44045ca43e785c24410cdc23d", "testCaseId": "d90981d44045ca43e785c24410cdc23d", "fullName": "tests.test_create_user.TestCreateUser#test_create_two_similar_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "Kotiki"}, {"name": "thread", "value": "15004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}